src/commands/apply/index.ts

```diff
@@ -46,7 +46,7 @@ export default class Apply extends Command {
   static flags = {
     space: Flags.string({ default: undefined, description: 'Space id', required: true }),
     environment: Flags.string({ default: undefined, description: 'Target environment id', required: true }),
-    file: Flags.string({ description: 'File path to changeset file', required: false, default: 'changeset.json' }),
+    file: Flags.string({ description: 'File path to changeset file', required: true, default: undefined }),
```

Comment:
Ahouldn't this be optional, and if not provided fall back to the new schema 🤔

Reply:
But this is on the apply and we can't fall back on the new schema as we don't know the specifics of the changeset file (e.g. timestamp, env names etc.)

Reply:
I missed the fact that this is `apply` 🙈